packages/yarnpkg-fslib/tests/npath.test.ts

```diff
@@ -75,6 +75,36 @@ describe(`Portable paths`, () => {
             const outputPath = `/C:`;
             expect(npath.toPortablePath(inputPath)).toEqual(outputPath);
           });
+
+          it(`should support UNC Windows paths (//[server]/[sharename]/)`, () => {
+            const inputPath = '\\\\Server01\\user\\docs\\Letter.txt';
+            const outputPath = `//Server01/user/docs/Letter.txt`;
```

Comment:
I'm not sure it would work - the `//` will be removed when going through `ppath.normalize`, which is called by all other path functions (`join`, etc). I think we'll need `/unc/<name>/...`

Reply:
Ah, right. I've totally forgotten about `ppath.normalize`.